src/contracts/signing-keys-registry/signing-keys-registry.service.ts

```diff
@@ -384,6 +314,7 @@ export class SigningKeyEventsCacheService {
       stakingModulesAddresses: cachedEvents.headers.stakingModulesAddresses,
       startBlock: cachedEvents.headers.startBlock,
       endBlock,
+      isValid,
```

Comment:
i dont think we should here the same as in deposit events cache
these two caches solve diff problems
we want to make pause even if events list is not valid
but we dont want to make unvetting if this chache is not valid

I see here two options:

return in this method only valid part of cache and don't  set isValid in header at all
if for duplicates identification we don't have enough data, we will process them as unresolved, as we already do
pluses: we will know for what keys we don't have events in cache, it will help partially resolve duplicates
unresolved list has the same meaning as isValid